dist/performance.js

```diff
@@ -210,15 +190,19 @@ function getGamingMetrics(rawDoc, lcp_elem_stats) {
 // Source: https://stackoverflow.com/questions/57786082/determine-how-much-of-the-viewport-is-covered-by-element-intersectionobserver
 // percentage is a whole number (ex: 90, not .9)
 function doesElementCoverPercentageOfViewport(element, percentage) {
-        const elementBCR = element.getBoundingClientRect();
-        const percentOfViewport = ((elementBCR.width * elementBCR.height) * calcOcclusion(elementBCR)) / ((window.innerWidth * window.innerHeight) / 100);
+        const percentOfViewport = getPercentOfViewport(element) * 100;
 
         if (percentOfViewport > percentage) {
             return true;
         }
         return false;
 }
 
+function getPercentOfViewport(element) {
+        const elementBCR = element.getBoundingClientRect();
+        return (elementBCR.width * elementBCR.height * calcOcclusion(elementBCR)) / (window.innerWidth * window.innerHeight);
+}
+
```

Comment:
I like the fix but am not loving the changing definition of percentage here:
- percentage: 0-100
- percentOfViewport: 0 -100
- getPercentOfViewport: 0 - 1

Should we move the multiplication into `getPercentOfViewport` to at least have consistency of 0-100 everywhere?

Reply:
I'll just make it decimals everywhere, the int stuff was just a convenience for the person who originally wrote the code